src/frontends/ir/tests/frontend_test_mmap.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/file_utils.hpp"
+#include "frontend_test.hpp"
+#include "openvino/opsets/opset1.hpp"
+
+class IRFrontendMMapTestsAdvanced : public ::testing::TestWithParam<bool>, public IRFrontendTestsImpl {
+protected:
+    size_t binsize;
+
+    void SetUp() override {
+        size_t CONST_SIZE = 10000000; /*~ 77 MB for f64 to guarantee size is more than size of mapped libraries*/
+        auto parameter = std::make_shared<ov::opset1::Parameter>(ov::element::f64, ov::Shape{CONST_SIZE});
+        parameter->set_friendly_name("input");
+        auto constant = std::make_shared<ov::opset1::Constant>(ov::element::f64,
+                                                               ov::Shape{CONST_SIZE},
+                                                               std::vector<double>(CONST_SIZE, 0));
+        constant->set_friendly_name("value1");
+        auto add = std::make_shared<ov::opset1::Add>(parameter, constant);
+        add->set_friendly_name("Add");
+        auto result = std::make_shared<ov::opset1::Result>(add);
+        result->set_friendly_name("output");
+        auto model = std::make_shared<ov::Model>(ov::NodeVector{result}, ov::ParameterVector{parameter});
+
+        auto filePrefix = CommonTestUtils::generateTestFilePrefix();
+        xmlFileName = filePrefix + "_IrFrontendTestModel.xml";
+        binFileName = filePrefix + "_IrFrontendTestModel.bin";
```

Comment:
Stop forming bin file name from xml file name. It is done automatically inside serialize if you leave bin_path with its default value.

Reply:
Done, thanks!